test/src/test/java/jenkins/I18nTest.java

```diff
@@ -61,4 +61,18 @@ public void test_valid() throws IOException, SAXException {
         JSONObject data = response.getJSONObject("data");
         Assert.assertEquals("Initialisiere Log-Rekorder", data.getString("LogRecorderManager.init"));
     }
+    @Test
+    public void test_valid_region() throws IOException, SAXException {
+        JSONObject response = jenkinsRule.getJSON("i18n/resourceBundle?baseName=hudson.logging.Messages&language=de-DE").getJSONObject();
+        Assert.assertEquals("ok", response.getString("status"));
+        JSONObject data = response.getJSONObject("data");
+        Assert.assertEquals("Initialisiere Log-Rekorder", data.getString("LogRecorderManager.init"));
+    }
+    @Test
+    public void test_valid_region_variant() throws IOException, SAXException {
+        JSONObject response = jenkinsRule.getJSON("i18n/resourceBundle?baseName=hudson.logging.Messages&language=de_DE_cloudBees").getJSONObject();
```

Comment:
Ideally there should be a test verifying all separators and their combinations. e.g. en_US|cloudBeers